cfme/automate/explorer/method.py

```diff
@@ -228,6 +229,37 @@ def read(self):
         return self.all_vars
 
 
+class EntryPoint(Text, ClickableMixin):
+    def fill(self, value):
+        """Used to select the automate method from automate domain"""
+        if value:
+            self.click()
+            self.parent_view.embedded_view.wait_displayed('15s')
+            self.parent_view.embedded_view.tree.click_path(*value)
+            return True
+        return None
+
+
+class EmbeddedView(View):
+    """This view is for embedding automate method"""
+    title = Text(
+        './/div[contains(@class, "modal-header ng-scope")]/h4[contains(@class, "modal-title")]'
+    )
+    include_domain = Text(
+        './/div[contains(@class, "bootstrap-switch-container")]/span[contains(@class, '
+        '"bootstrap-switch-label")]'
+    )
+    close = Text('.//button[contains(@ng-click, "closeModal")]')
+    tree = ManageIQTree("treeview-entrypoint_selection")
+
+    @property
+    def is_displayed(self):
+        return (self.title.text == "Select Item" and
+                self.tree.has_path("Datastore", "RedHat (Locked)") and
```

Comment:
I think these strings should be constructed from the `appliance.product_name` attribute, since its related to productization of ManageIQ.

Will they always be `(Locked)` or is that state dependent?

Reply:
Yes. They always be locked as those are default domains.